packages/error-reporting/package.json

```diff
@@ -4,7 +4,7 @@
   "main": "./src/index.js",
   "repository": "GoogleCloudPlatform/google-cloud-node",
   "scripts": {
-    "test": "nyc --exclude=\"fuzzer.js\" mocha ./test/unit/**/*.js",
+    "test": "nyc --exclude=\"fuzzer.js\" mocha ./test/unit/*.js ./test/unit/**/*.js",
```

Comment:
With the new pattern matching expression, you should be able to remove `./test/unit/*.js`